vizro-core/src/vizro/managers/_data_manager.py

```diff
@@ -3,74 +3,50 @@
 from __future__ import annotations
 
 import logging
+import os
 import warnings
-from typing import Any, Callable, Dict, Optional, Protocol, Union
+from functools import partial
+from typing import Callable, Dict, Optional, Union
 
 import pandas as pd
 import wrapt
 from flask_caching import Cache
 
 from vizro.managers._managers_utils import _state_modifier
 
-logger = logging.getLogger(__name__)
+
```

Comment:
@petar-qb as with https://github.com/mckinsey/vizro/pull/479 I suggest you run an example app _without_ the following environment variables that are set in our hatch default environment:
```
DASH_DEBUG = "true"
VIZRO_LOG_LEVEL = "DEBUG"
```

That will make the logs clearer and easier to understand.